src/community/clustering.jl

```diff
@@ -26,29 +26,52 @@ the maximum number of possible triangles, respectively, for each node in the lis
 
 This function is related to the local clustering coefficient `r` by ``r=\frac{a}{b}``.
 """
```

Comment:
I think this docstring needs to be moved to the actual `local_clustering()` function, not be above the mutable one.